### PR TITLE
Issue 21

### PR DESCRIPTION
**Pull request by parkjh688**
review 부탁드립니다.

### CODE REVIEW

**Comment by parkjh688** [comment]
**Regularization** 부분이 어색할것 같아요. 
번역하면서 긴가민가해서ㅋㅋㅋ

File: 21_Tensorflow와 Keras를 활용한 가상화폐 가격 예측하기.md [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+model.add(Dropout(0.5))
+model.add(Conv1D( strides=2, filters=nb_features, kernel_size=2))
+'''
+model.compile(loss='mse', optimizer='adam')
+model.fit(training_datas, training_labels,verbose=1, batch_size=batch_size,validation_data=(validation_datas,validation_labels), epochs = epochs, callbacks=[CSVLogger(output_file_name+'.csv', append=True),ModelCheckpoint('weights/'+output_file_name+'-{epoch:02d}-{val_loss:.5f}.hdf5', monitor='val_loss', verbose=1,mode='min')])
+```
+
+<br></br>
+
+내가 빌드한 첫 번째 모델은 CNN 입니다. 위의 코드에서는 사용할 GPU 개수를 "1"로 설정합니다(저는 4개의 GPU를 가지고 있지만, 당신은 원하는 만큼의 GPU를 상용할 수 있습니다). 왜냐하면 Tensorflow로 여러개의 GPUs를 사용하는 것은 그다지 잘 돌아가지 않기 때문에, 사용할 GPU를 1개로 제한하는 것이 더 현명한 방법일지도 모릅니다. 당신이 GPU가 없다고해도 걱정하지마세요. GPU를 설정하는 코드를 가뿐히 무시하기만 하면 됩니다.
```
Comment (insurgent92) [inline 21_Tensorflow와 Keras를 활용한 가상화폐 가격 예측하기.md]: type 상용 -> 사용

File: 21_Tensorflow와 Keras를 활용한 가상화폐 가격 예측하기.md [hunk 2]
```diff
@@ -0,0 +1,750 @@
...
+
+<br></br>
+
+비록 예측된 결과가 좋아보여도, overfittinng(과적합)의 걱정이 있습니다. LSTM을 LeakyReLU로 학습할 때의 loss(손실)과 validation(검증)의 loss에는 갭이 있습니다(5.97E-06 vs 3.92E-05). 이 손실을 최소화 하기 위해서 Regularization(정규화)가 필요합니다.
+
+<br></br>
+
+#### Regularization
+
+가장 좋은 regularization 전략을 찾기 위해서, 저는 L1 regularization, L2 regularization 으로 몇 개의 실험을 했습니다. 첫 번쨰로 우리는 LSTM에 데이터를 쉽게 피팅시킬 수 있는 새로운 기능을 정의해야 합니다. 저는 bias를 벡터에 대한 regularization인 bias regularizer를 예로 들겠습니다.
```
Comment (insurgent92) [inline 21_Tensorflow와 Keras를 활용한 가상화폐 가격 예측하기.md]: typo 첫 번쨰 -> 첫 번째

**Comment by parkjh688** [comment]
넵 확인했습니다.
